src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -39,6 +39,9 @@ public boolean start() throws Exception {
         System.out.println(templates);
 
         Node node = computer.getNode();
+        if (node == null) {
+            throw new Exception("Computer does not correspond to a live node");
```

Comment:
On the getNode() method, it says 

> Returns:
> null if the configuration has changed and the node is removed, yet the corresponding Computer is not yet gone.

I don't know exactly what the user can do. I see this as a corner case scenario.
This null check and exception I've used them exactly how they are used in the ws step implementation: https://github.com/jenkinsci/pipeline-plugin/blob/1.0/support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/WorkspaceStepExecution.java#L38